db/migrations/20231114130216_add_missing_fk_indexes.rb

```diff
@@ -0,0 +1,61 @@
+Sequel.migration do
+  up do
+    if database_type == :postgres
+      add_index :organizations, :quota_definition_id, name: :organizations_quota_definition_id_index, options: [:concurrently]
```

Comment:
The current way this creates indices does create one large transaction. With concurrently this might not be a problem since nothing is locked and on mysql there are implcit commits after every DDL statement. So i would be fine leaving it in a transaction.

One thing i would change for resilience is to just make the add_index call when no index exists yet thus making the up migration idempotent. In case of unknown errors while creating the index, the migration is not stuck in a state where neither the down nor the up part can be executed anymore and a operator needs to manipulate the db.

Either way one can look at https://github.com/cloudfoundry/cloud_controller_ng/blob/main/db/migrations/20230725110800_restructure_index_for_improved_pagination_order.rb#L553-L561 where there is one transaction per index and also the up and down parts are idempotent